docs/development/guide.md

```diff
@@ -79,13 +81,11 @@ See the rules for naming functions and components, and the architecture conventi
 - Each line should contain no more than one statement.
 - Always put the opening brace in the same line as the previous statement.
 - Avoid using the ternary operator (`statement ? true : false`). Use the guard (`&&`) instead.
-- Do not use a prefix or a suffix on an interface like `IVariable` or `VariableInterface`. That does not apply to `AsyncApiInterface`, `ConfigInterface`, and `ThemeInterface`.
+- Do not use a prefix or a suffix on an interface like `IVariable` or `VariableInterface`. That does not apply to `AsyncApiInterface` and `ConfigInterface`.
```

Comment:
I thought `AsyncApiInterface` goes away. At least you removed it from main readme

Reply:
Good catch! I forgot about that 😅